ii/Views/Manager/RuleSetFileNode.cs

```diff
@@ -35,7 +35,7 @@ public RuleSet GetRuleSet()
         var yaml = System.IO.File.ReadAllText(File.FullName);
 
         var deserializer = IsIdentifiableAbstractRunner.GetDeserializer();
-        return _ruleSet = deserializer.Deserialize<RuleSet>(yaml);
+        return _ruleSet = deserializer.Deserialize<RuleSet>(yaml) ?? new RuleSet();
```

Comment:
This is a behaviour change I think, if users ops setings file points to an empty file we now dodge a null reference and show an empty node in UI.  User may be able to add nodes later on or copy into that file and serialization/save should work.

We could just throw here instead though, thats an option.